internal/utility.go

```diff
@@ -108,7 +108,6 @@ func getMaxUploadDiskConcurrency() int {
 	return GetMaxConcurrency("WALG_UPLOAD_DISK_CONCURRENCY", 1)
 }
 
-// TODO : unit tests
```

Comment:
There left one case you don't cover. os.SetEnv("WALG_UPLOAD_CONCURRENCY", "incorrect number").
But interface of this function is sick because we shouldn't to Fatal there with an error.
Let's don't do such things as fatal or os.Exit here and just return an error to the main.